server/queue/queue-api/src/test/java/org/apache/james/queue/api/ManageableMailQueueContract.java

```diff
@@ -431,7 +431,7 @@ default void removeByRecipientShouldRemoveSpecificEmailWhenMultipleRecipients()
         assertThat(getManageableMailQueue().browse())
             .extracting(ManageableMailQueue.MailQueueItemView::getMail)
             .extracting(Mail::getName)
-            .containsExactly("name1");
+            .containsExactly("name2");
```

Comment:
why it didn't break before ?

Reply:
Because this test is:
 - not implemented for files
 - Failing thus disabled for JMS based queue (none of the mail is removed)
 - Newly introduced for memory